src/components/ContactList/ContactList.scss

```diff
@@ -0,0 +1,17 @@
+.contact-list {
+
+}
+
+.contact-list li {
```

Comment:
Try never to use tags as selectors. Add classes using BEM.
Revise all your code.